requirements/dev.txt

```diff
@@ -5,6 +5,7 @@
 tox==3.19.0
 pytest==6.0.1
 WebTest==2.0.35
+pandas==1.1.4
```

Comment:
Skoro bych i přidal komentář, k čemu že se to používá.

Reply:
Hotovo